ubyssey/templates/components/event-box.html

```diff
@@ -1,15 +1,13 @@
-<div class="c-event c-event--box{% if size %} c-event--{{ size}}{% endif %}">
+<div onclick=location.href="{% url 'event' event.pk.hex %}"; style="cursor: pointer;" class="c-event c-event--box{% if size %} c-event--{{ size}}{% endif %}">

```

Comment:
Instead of using `onClick`, it'd be better to change the `div` to an `a` tag and use `href` as you normally would.